src/components/HelpSection.vue

```diff
@@ -289,6 +261,20 @@ export default {
 }
 </script>
 <style scoped>
+    .help-link {
+        padding-bottom: 8px;
+    }
+    .help-link .help-link-a {
+        color: #ED592F;
+    }
+    .help-links .help-link .help-link-a:hover {
+        font-weight: bold;
```

Comment:
Changing the font weight to bold looks pretty bad IMO, and doesn't match the hover state on other links where color changes, but font weight is kept the same.

Reply:
I agree :) Changed both the font weight and the color.